stdlib/public/runtime/Float16Support.cpp

```diff
@@ -27,7 +27,10 @@
 //
 //===----------------------------------------------------------------------===//
 
-#if defined(__x86_64__) && !defined(__APPLE__)
+// Android NDK <r21 do not provide `__aeabi_d2h` in the compiler runtime,
+// provide shims in that case.
+#if (defined(ANDROID) && defined(__ARM_ARCH_7A__) && defined(__ARM_EABI__)) || \
+  (defined(__x86_64__) && !defined(__APPLE__))
 
 #include "../SwiftShims/Visibility.h"
 
```

Comment:
You don't need h2f and f2h on Android, right?

Reply:
Im running a build on Azure - hopefully its not needed.  Otherwise, I think that we may need to do something a bit more invasive.

- `__aeabi_d2f`
- `__aeabi_f2d`
- `__aeabi_h2f`
- `__aeabi_f2h`
- `__aeabi_d2h`

Is the set of functions in this family.

Reply:
Ok, let's see what your build turns up.